Yoast/javascript
# P2-59 Restore insights and internal linking functionality.

## Summary


This PR can be summarized in the following changelog entry:

* Restores changes to the JavaScript files that were made for improved internal linking. they somehow got lost between merging the feature branch in develop and creating the latest RC.

## Relevant technical choices:

* 

## Test instructions

This PR can be tested by following these steps:

* Open a post with more than 300 words of content in the post editor.
* You should see an insights collapsible in the YoastSEO metabox.
* You should see an insights collapsible in the YoastSEO sidebar.
* You should see an internal linking suggestions collapsible in the YoastSEO sidebar.
* Open the browser console, you should not see the error as described in the issue:
![image](https://user-images.githubusercontent.com/10195175/85846439-1ef44300-b7a6-11ea-80ed-d7f97812c6b4.png)
* Please note: during development we noticed other JavaScript errors that seem to be unrelated to this issue.

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes [P2-59](https://yoast.atlassian.net/browse/P2-59)
